hopdrops/hopdrops
# Add Navigation For Onboarding

# Add Navigation For Onboarding
> Issue #84 

This allows the user to navigate between the configuration pages during onboard. Currently, it works as intended, but I would like to add a few tests on the navigation component. Since I'll be on vacation, I figure it would be a good idea to test it and make sure it doesn't cause any potential build or rendering issues.

PR Include:
- Reusable nav component for Onboarding
- Minor stylistic changes on Location
- Start of Category Page (in order to test navigation)
- Add A Site Title For Now - Hopdrops - "Live Can Releases for Your Loyal Fans"

![image](https://user-images.githubusercontent.com/6020066/63669502-0fbff580-c78f-11e9-9635-e0e9a55a9227.png)

![image](https://user-images.githubusercontent.com/6020066/63668714-bbb41180-c78c-11e9-9556-47d4dc82f0e3.png)

> Note: This page isn't complete yet

![image](https://user-images.githubusercontent.com/6020066/63669133-1a2dbf80-c78e-11e9-9f0e-439383a73a18.png)

## Testing
1. `cd /app`
2. `npm start -s`
3. Go to Browser:  `localhost:3000/settings/locations`
4. Select locations and hit the next button
5. An incomplete category page will render
6. Select `back` and you should return back to the locations view
